snippets/facets.liquid

```diff
@@ -1051,7 +928,7 @@
                 {%- else -%}
                   {{ 0 | money }}
                 {%- endif -%}
-                -
+                -&nbsp;
                 {%- if filter.max_value.value -%}
                   {{ filter.max_value.value | money }}
                 {%- else -%}
```

Comment:
These if-statements could be replaced with:
```
{{ filter.min_value.value | default: 0 | money }} - {{ filter.max_value.value | default: filter.range_max | money }}
```

I know it's longer, but I think it's clear what it does. And you don't need to manage the whitespace with `&nbsp;`